modules/checkout/commerce_checkout.info.yml

```diff
@@ -10,3 +10,8 @@ dependencies:
 config_devel:
   install:
     - commerce_checkout.commerce_checkout_flow.default
+    - core.entity_view_display.commerce_order.default.summary
```

Comment:
Shoot; I forgot our discussion - are we porting the order total footer here? or removing, then porting and adding in once this is closed a a follow up